clustered/client/src/test/java/org/ehcache/clustered/client/docs/GettingStarted.java

```diff
@@ -211,4 +211,47 @@ public void loadDocsXml() throws Exception {
     new XmlConfiguration(getClass().getResource("/configs/docs/ehcache-clustered.xml"));
   }
 
+  @Test
+  public void unknownClusteredCacheExample()
+  {
+    // tag::unknownClusteredCacheExample[]
+
+    CacheManagerBuilder<PersistentCacheManager> cacheManagerBuilderAutoCreate = CacheManagerBuilder.newCacheManagerBuilder()
+            .with(ClusteringServiceConfigurationBuilder.cluster(URI.create("terracotta://localhost:9510/my-application"))
+                .autoCreate()  // <1>
+                .resourcePool("resource-pool", 32, MemoryUnit.MB, "primary-server-resource"));
+
+    final PersistentCacheManager cacheManager1 = cacheManagerBuilderAutoCreate.build(false);
+    cacheManager1.init();
+
+    CacheConfiguration<Long, String> cacheConfigDedicated = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class,
+    ResourcePoolsBuilder.newResourcePoolsBuilder()
+        .with(ClusteredResourcePoolBuilder.clusteredDedicated("primary-server-resource", 8, MemoryUnit.MB)))  // <2>
+    .add(ClusteredStoreConfigurationBuilder.withConsistency(Consistency.STRONG))
+    .build();
+
+    Cache<Long, String> cacheDedicated = cacheManager1.createCache("my-dedicated-cache", cacheConfigDedicated);  // <3>
+
+    CacheManagerBuilder<PersistentCacheManager> cacheManagerBuilderExpecting = CacheManagerBuilder.newCacheManagerBuilder()
+            .with(ClusteringServiceConfigurationBuilder.cluster(URI.create("terracotta://localhost:9510/my-application"))
+                .expecting()  // <4>
+                .resourcePool("resource-pool", 32, MemoryUnit.MB, "primary-server-resource"));
+
+    final PersistentCacheManager cacheManager2 = cacheManagerBuilderExpecting.build(false);
+    cacheManager2.init();
+
+    CacheConfiguration<Long, String> cacheConfigUnknown = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class,
```

Comment:
actually this does sound way better, and makes more sense.  I will get rid of unknown and replace with unspecified.